### ISSUE TITLE
drtop down instead of scroll ?

### ISSUE THREAD

**Issue by tahoedan** [MrOtherGuy/firefox-csshacks#175]
How can i adjust this so that any bookmarks over the shown rows are in a drop down rather than scroll ?

**Comment by MrOtherGuy** [comment]
You mean multi-row_bookmarks.css? Sadly I don't think that would work.

Can't say I'm absolutely certain, but I'm very confident that this won't be possible.

**Comment by tahoedan** [comment]
> You mean multi-row_bookmarks.css? Sadly I don't think that would work.
> 
> Can't say I'm absolutely certain, but I'm very confident that this won't be possible.

Yes i was talking about your multi row css.     Thank you for the response.
